src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const styleArray = sourceString.split(';');
+  const removingSpaces = styleArray.map((lines) => lines.trim());
+  const filterValues = removingSpaces.filter((elem) => elem !== '');
+
+  filterValues.forEach((elem) => {
+    const [key, value] = elem.split(':');
```

Comment:
When splitting the `elem` by ':', it's safer to use destructuring with a default value for `value` to handle cases where a colon might be missing in the style string. For example: `const [key, value = ''] = elem.split(':');`. This ensures that `value` is an empty string rather than `undefined` if the colon is missing.